src/components/game/item.ts

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+import item from "../../../resources/images/info-bg.png";
```

Comment:
Why is it a png? 🤔 
Moreover, it should go to the bottom of the page
<img width="451" alt="image" src="https://user-images.githubusercontent.com/45454460/199450183-01990f0d-c9d0-4621-a8a5-d4e8225c8e2c.png">